src/commands/General/latex.ts

```diff
@@ -63,6 +98,63 @@ export class LatexCommand extends HorizonCommand<typeof config> {
       replyTo = submit;
       equation = submit.fields.getTextInputValue('equation');
     }
-    await replyTo.reply(settings.apis.latex + encodeURIComponent(equation));
+
+    try {
+      await this._tryToGenerate(replyTo, equation, 0);
+    } catch (e) {
+      if (e instanceof EvalError) {
+        await replyTo.reply({
+          content: "Ça n'a pas l'air d'être une équation valide ! Le message d'erreur LaTeX est :\n"
+            + `\`${e.message}\``,
+          ephemeral: true,
+        });
+      } else {
+        await replyTo.reply({
+          content: "L'image de la formule n'a pas pu être générée.",
+          ephemeral: true,
+        });
```

Comment:
Il faudrait les passer dans la configuration.
Le deuxième est straightforward, pour le premier faudra utiliser `pupa`. je te laisse regarder comment est-ce qu'on fait dans d'autres commandes, mais il faudrait en gros rajouter deux entrées dans [l'objet de config](https://github.com/horizon-efrei/HorizonBot/blob/bcda527400e960b4b3e91eaaf8a3eb0c15793ca2/config/commands/general.ts#L71-L72), puis ici faire un `this.messages.{le nom de l'entrée}` (l'autocomplétion TypeScript pourra bien t'aider).

Exemple dans le fichier de la commande ping: [commande](https://github.com/horizon-efrei/HorizonBot/blob/bcda527400e960b4b3e91eaaf8a3eb0c15793ca2/src/commands/General/ping.ts#L26) - [config](https://github.com/horizon-efrei/HorizonBot/blob/bcda527400e960b4b3e91eaaf8a3eb0c15793ca2/config/commands/general.ts#L98C62-L98C62)